lib/async_hooks.js

```diff
@@ -359,9 +354,9 @@ function emitHookFactory(symbol, name) {
     } catch (e) {
       fatalError(e);
     }
-    processing_hook = false;
+    processing_hook -= 1;
```

Comment:
I guess, but there is no way to escape `fatalError` so the code will never be executed.

```js
try {
  throw new Error();
} catch (e) {
  console.log('catch');
} finally {
  console.log('finally');
}
```

outputs `catch finally`.